src/js/components/TimeAgo.js

```diff
@@ -73,7 +73,7 @@ class TimeAgo extends React.Component {
       <time
         title={DateUtil.msToUTCDate(time)}
         dateTime={DateUtil.msToUTCDate(time)}
-        {...Util.omit(this.props, ["prefix", "suppressSuffix", "time"])}
+        {...Util.omit(this.props, Object.keys(TimeAgo.propTypes))}
```

Comment:
Isn't this kind of strange removing all the `propTypes` from the read-only `props`? If this always returns `{}` why even have this line?